pkg/api/testing/defaulting_test.go

```diff
@@ -168,6 +168,8 @@ func TestDefaulting(t *testing.T) {
 		{Group: "scheduling.k8s.io", Version: "v1alpha1", Kind: "PriorityClassList"}:                            {},
 		{Group: "scheduling.k8s.io", Version: "v1beta1", Kind: "PriorityClassList"}:                             {},
 		{Group: "scheduling.k8s.io", Version: "v1", Kind: "PriorityClassList"}:                                  {},
+		{Group: "flowcontrol.apiserver.k8s.io", Version: "v1alpha1", Kind: "FlowSchema"}:                        {},
+		{Group: "flowcontrol.apiserver.k8s.io", Version: "v1alpha1", Kind: "FlowSchemaList"}:                    {},
```

Comment:
We have defaulting in v1beta1 too, right?

Reply:
fixed